pkg/azuredisk/nodeserver_test.go

```diff
@@ -893,3 +893,31 @@ func TestGetDevicePathWithLUN(t *testing.T) {
 		}
 	}
 }
+
+func TestGetDevicePathWithMountPath(t *testing.T) {
+	d, _ := NewFakeDriver(t)
+	err := "exit status 1"
+	mountPath := "unit-test"
+
+	if runtime.GOOS == "darwin" {
+		err = "executable file not found in $PATH"
+	}
+
+	tests := []struct {
+		desc        string
+		req         string
+		expectedErr error
+	}{
+		{
+			desc:        "Invalid device path",
```

Comment:
could you also add a valid test, e.g. `/sys`
```console
findmnt -o source --noheadings --mountpoint /sys
sysfs
```

Reply:
Is there a mount path equivalent for running on Mac OS? I have been looking but haven't had any luck